internal/app/machined/pkg/controllers/runtime/kernel_param_defaults.go

```diff
@@ -120,6 +120,11 @@ func (ctrl *KernelParamDefaultsController) getKernelParams() []*kernel.Param {
 			Key:   "proc.sys.net.ipv4.tcp_keepalive_intvl",
 			Value: "60",
 		},
+		// Consider fwmark for rp_filter routing table lookup.
+		{
+			Key:   "proc.sys.net.ipv4.conf.kubespan.src_valid_mark",
+			Value: "1",
+		},
```

Comment:
We might even consider enabling rp_filter on all interfaces by default as Red Hat and SUSE do for increased security against IP spoofing

Reply:
Nit: here I can + the `constants.KubeSpanLinkName` I guess

Reply:
I guess we can set to 1 as default like RHEL

Reply:
Let's discuss that later for 1.10, it's not to be backported like this change is